ts/components/ConversationListItem.tsx

```diff
@@ -157,7 +157,7 @@ export class ConversationListItem extends React.PureComponent<Props> {
 
     return (
       <div className="module-conversation-list-item__message">
-        <div
+        <div dir="auto"
```

Comment:
Can we make sure this attribute is on the next line? If you enable prettier in your editor the formatting should be automatic.

Reply:
Like this https://github.com/sha-265/Signal-Desktop/commit/b67cebfb5ec30f4350c1a1268e4f67e3e845e7f5?

Reply:
Yep!